src/pages/layer-2.tsx

```diff
@@ -40,153 +51,92 @@ import { getImage } from "../utils/image"
-`
-
-const FlexContainer = styled.div<{ flexPercent: string | number }>`
-  flex: ${(props) => props.flexPercent}%;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex: 100%;
-  }
-`
-
-const Flex50 = styled.div`
-  flex: 50%;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex: 100%;
-  }
-`
-
-const StyledIcon = styled(Icon)`
-  fill: ${(props) => props.theme.colors.text};
-  margin-right: 0.5rem;
-  opacity: 0.8;
-  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
-  }
-  &:hover,
-  &:active,
-  &:focus {
-    fill: ${({ theme }) => theme.colors.primary};
-  }
-`
+type ChildOnlyType = {
+  children: ReactNode
+}
 
-const TwoColumnContent = styled.div`
-  display: flex;
-  justify-content: space-between;
-  gap: 4rem;
-  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
-    flex-direction: column;
-    align-items: flex-start;
-    margin-left: 0rem;
-    margin-right: 0rem;
+const ContentBox = (
+  props: HTMLAttributes<"div"> & {
+    children: ReactNode
+    isLightGrayBg?: boolean
   }
-`
+) => (
+  <Box
+    px={8}
+    py={12}
+    width="full"
+    {...(props.isLightGrayBg && { background: "layer2ContentSecondary" })}
+  >
+    {props.children}
+  </Box>
+)
 
-const InfoGrid = styled(CardGrid)`
-  display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(min(100%, 340px), 1fr));
-  gap: 1rem 2rem;
-  & > div {
-    height: fit-content;
-    &:hover {
-      transition: 0.1s;
-      transform: scale(1.01);
-      img {
-        transition: 0.1s;
-        transform: scale(1.1);
-      }
-    }
-  }
-`
+const StyledInfoIcon = () => (
+  <Icon
+    as={MdInfoOutline}
+    color="text"
+    mr={2}
+    opacity={0.8}
+    _hover={{ color: "primary" }}
+    _active={{ color: "primary" }}
+    _focus={{ color: "primary" }}
+  />
+)
 
-const RollupCard = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  background: ${(props) => props.theme.colors.ednBackground};
-  border-radius: 2px;
-  border: 1px solid ${(props) => props.theme.colors.lightBorder};
-  padding: 1.5rem;
-  flex: 50%;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex: 100%;
-  }
-`
+const TwoColumnContent = (props: ChildOnlyType) => (
+  <Flex
+    justifyContent="space-between"
+    gap={16}
+    flexDirection={{ base: "column", lg: "row" }}
+    alignItems={{ base: "flex-start", lg: "normal" }}
+    {...props}
+  />
+)
 
-const StatsContainer = styled.div`
-  display: flex;
-  justify-content: center;
-  align-content: center;
-  margin-bottom: 4rem;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex-direction: column;
-  }
-`
+const StatDivider = () => {
+  const responsiveOrientation = useBreakpointValue<DividerProps["orientation"]>(
+    { base: "horizontal", md: "vertical" }
+  )
+  return (
+    <Divider
+      orientation={responsiveOrientation}
```

Comment:
Probably you've already explained this to me xD but why we can't just do
```suggestion
      orientation={{ base: "horizontal", md: "vertical" }}
```

Reply:
Also, I noted this bug when you load the page on desktop res:

https://user-images.githubusercontent.com/468158/219204060-5648b762-b51b-4d3f-8a77-4fbf6c3244bf.mp4

- we don't have that horizontal divider on high resolutions
- the vertical divider do not display

Reply:
@pettinarip

- To the first: What you are suggesting is creating a responsive object. Unfortunately, the `orientation` prop does not accept this, only strictly typed string values of `horizontal` or `vertical` if not `undefined`. `useBreakpointValue` will return a dynamic string based on breakpoint to be passed in and pass the type check.
- To the second: What is missed for the divider to work vertically with `height='100%` is the parent container receiving an explicit height. In this case, it looks like the height can be set to `100px` above the `md` breakpoint and the layout should still work.